op-node/rollup/driver/origin_selector.go

```diff
@@ -47,6 +47,12 @@ func (los *L1OriginSelector) FindL1Origin(ctx context.Context, l1Head eth.L1Bloc
 		return eth.L1BlockRef{}, err
 	}
 
+	// We know there is at least one block & we have hit the sequencer drift, we need to advance.
+	if l2Head.Time+los.cfg.BlockTime > currentOrigin.Time+los.cfg.MaxSequencerDrift {
+		log.Info("Must try to advance origin") // TODO: Full log
```

Comment:
Want to knock out this todo before merge?

Reply:
yea, good catch